sample.lua

```diff
@@ -13,12 +13,23 @@ cmd:option('-temperature', 1)
 cmd:option('-gpu', 0)
 cmd:option('-gpu_backend', 'cuda')
 cmd:option('-verbose', 0)
+cmd:option('-seed', 0)
 local opt = cmd:parse(arg)
 
 
 local checkpoint = torch.load(opt.checkpoint)
 local model = checkpoint.model
 
+if opt.seed == 0 then
```

Comment:
What is someone wants to use a `-seed 0`? Is it possible to check if a flag is explicitly set? If it is, then you should change the `if` conditional to it; if not, then please document that the seed cannot be zero.

Reply:
I don't think this is possible (but these are my first ever lines of lua), the lua command line structure sets it to a default if omitted, so it's kind of set by definition. Will document that behavior of including the flag with 0 is the same as omitting the flag rather than explicitly setting the seed to 0.